spark/components/buttons/vanilla/button.stories.js

```diff
@@ -1,48 +1,69 @@
-import { withKnobs, text } from '@storybook/addon-knobs';
 
 export default {
   title: 'Components|Buttons',
-  decorators: [withKnobs],
 };
 
 export const primary = () => (
   `
     <button class="sprk-c-Button" data-id="button-1">
-      ${text('button text', 'Button')}
+      Button
     </button>
   `
 );
 
 export const secondary = () => (
-  '<button class="sprk-c-Button sprk-c-Button--secondary" type="button" data-id="button-1">Button</button>'
+  `
+    <button class="sprk-c-Button sprk-c-Button--secondary" type="button" data-id="button-1">
+    Button
+    </button>
+  `
 );
 
 export const tertiary = () => (
-  '<button class="sprk-c-Button sprk-c-Button--tertiary" data-id="button-1">Button</button>'
+  `
+    <button class="sprk-c-Button sprk-c-Button--tertiary" type="button" data-id="button-2">
+    Button
+    </button>
+  `
 );
 
 export const disabled = () => (
-  '<button class="sprk-c-Button sprk-is-Disabled" data-id="button-1" disabled>Button</button>'
+  `
+  <button class="sprk-c-Button sprk-is-Disabled" data-id="button-1" disabled>
```

Comment:
@RVMendoza , this one has the same data-id as the secondary button and needs to be unique.